tests/unit_tests/test_wave_06.py

```diff
@@ -194,7 +212,13 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
+    assert result == False
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_c]
+    assert jesse.inventory == [item_d, item_e, item_f]
```

Comment:
Asserts with `==` will fail if the two lists have the same elements but not in the same order. It would be better to assert the individual items are in the list.